README.md

```diff
@@ -5,7 +5,7 @@
 [![build](https://github.com/purescript/spago/actions/workflows/build.yml/badge.svg)](https://github.com/purescript/spago/actions/workflows/build.yml)
 [![Maintainer: f-f](https://img.shields.io/badge/maintainer-f%2d-f-teal.svg)](http://github.com/f-f)
 
-*(IPA: /ˈspaɡo/)*
+_(IPA: /ˈspaɡo/)_
```

Comment:
Sorry, I updated this file for the use of `execArgs` -> `exec_args` and, as I use Prettier, it was auto-formatted. If you want me to revert the changes to this file then I can (though I personally agree with the changes made).

Reply:
Yeah it's fine, these changes are fairly innocent